### PR TITLE
Adds around 30 new ambience sounds, and changes ambience code a little bit

### PR DESCRIPTION
**Pull request by ghost**
:cl: Floyd / Qustinnus (And all the sounds by Kayozz11 from Yogstation)
soundadd: Adds about 30 new ambience sounds
refactor: added new defines for ambience lists.
/:cl:

I like sounds, TG needs more of them, and so, I will be adding a fuckton more. First round I already had ready; a ton of ambience sounds. Kayozz11 from Yogstation made these a few months back and I added them to yog for him; now I'm bringing them here.

I also added a way to easily categorize an area to make it use a specific group of ambient sounds, ambient_type. you can use it to set an area to ambient_type = "engineering" for example, and it'll set ambientsounds to a group of engineering themed ambient sounds.

Listen to the ambient sounds below! (Some might be missing or named differently in the PR, but most are the same)

https://www.youtube.com/watch?v=JZ5KzZvae7M

also I plan on adding a lot more sounds to things like machinery too, so while this PR is open feel free to share some feedback whether I should add sounds to machinery and other various objects in the game or not.

### CODE REVIEW

**Comment by Qustinnus** [comment]
im really tired so i probably messed something up :^)

File: code/game/area/Space_Station_13_areas.dm [hunk 1]
```diff
@@ -41,7 +41,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	power_environ = FALSE
 	valid_territory = FALSE
 	outdoors = TRUE
-	ambientsounds = list('sound/ambience/ambispace.ogg','sound/ambience/title2.ogg')
+	ambience_type = "space"
```
Comment (ChangelingRain) [inline code/game/area/Space_Station_13_areas.dm]: these should be defines

Comment (Qustinnus) [reply code/game/area/Space_Station_13_areas.dm]: Alright, will fix

**Comment by PKPenguin321** [comment]
Are these the russian station ambiance sounds? They're really good

**Comment by Qustinnus** [comment]
No, these are all original sounds made by Kayozz :P

**Comment by Qustinnus** [comment]
(who is a yog admin)

File: code/game/area/areas/centcom.dm [hunk 2]
```diff
@@ -96,6 +96,7 @@
 	noteleport = TRUE
 	blob_allowed = FALSE //Not... entirely sure this will ever come up... but if the bus makes blobs AND ops, it shouldn't aim for the ops to win.
 	flags_1 = NONE
+	ambience_type = "syndicate"
```
Comment (ShizCalev) [inline code/game/area/areas/centcom.dm]: where is this type defined?

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/game/area/Space_Station_13_areas.dm [hunk 3]
```diff
@@ -858,6 +854,7 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	name = "Teleporter Room"
 	icon_state = "teleporter"
 	music = "signal"
```
Comment (ShizCalev) [inline code/game/area/Space_Station_13_areas.dm]: Does each subtype of /area/teleporter need `music = "signal"`, or can that be cleaned up?

File: code/game/area/areas.dm [hunk 4]
```diff
@@ -107,6 +103,51 @@ GLOBAL_LIST_EMPTY(teleportlocs)
 	uid = ++global_uid
 	related = list(src)
 	map_name = name // Save the initial (the name set in the map) name of the area.
+	if(!ambientsounds)
```
Comment (AnturK) [inline code/game/area/areas.dm]: Put this in a wrapper. 

Also i think expanding the tags into list values would be more useful here. 
As in `list(HOLY,'addtionalsound.ogg')` so you can expand on the defaults more easily.

**Review by AnturK — COMMENTED** [review]
Thought this actually kills the badcode of actual ambience execution but oh well, this is fine.

**Comment by AnonymousNow** [comment]
Cute. The old ambience noises are something that helped me get into the feeling of the game in the first place. These work pretty well, by what I can hear.

**Comment by The-clown-known-as-Pepe** [comment]
AmbiAtmos, AmbiDanger is slowed down/distorted part of FTL track.
>original sounds
I'd imagine the rest is just someones existing work that was just distorted.

**Comment by Qustinnus** [comment]
They sound similar but you can easily tell they're not the same, go take your shitposting somewhere else, loser.

File: code/game/area/areas/centcom.dm (hunk 2 above)
Comment (Qustinnus) [reply code/game/area/areas/centcom.dm]: Good catch, thanks.

File: code/game/area/Space_Station_13_areas.dm (hunk 3 above)
Comment (Qustinnus) [reply code/game/area/Space_Station_13_areas.dm]: Doubt it, a lot of code seemed to have copy pasta variables.  I'll go remove some of that

**Comment by Qustinnus** [comment]
@anturK I wouldn't mind fully changing area's ambience code. But I wouldn't be sure what type of changes could really improve it.

**Review by ShizCalev — APPROVED** [review]

**Pull request merged**